### PR TITLE
refs #10970 updated lib tests

### PR DESCRIPTION
**Pull request by johnpmitsch**

### CODE REVIEW

File: app/models/katello/glue/pulp/pulp_content_unit.rb [hunk 1]
```diff
@@ -32,6 +32,7 @@ def with_identifiers(ids)
       end
 
       def in_repositories(repos)
+        repos = Array(repos)
```
Comment (komidore64) [inline app/models/katello/glue/pulp/pulp_content_unit.rb]: will this blow up anything if `repos` already happens to be an array?

Comment (johnpmitsch) [inline app/models/katello/glue/pulp/pulp_content_unit.rb]: Array() will return the same array if its passed an array, and a blank array if passed nil

Comment (komidore64) [inline app/models/katello/glue/pulp/pulp_content_unit.rb]: awesome!

**Comment by komidore64** [comment]
looks sane to me!

ACK

**Comment by johnpmitsch** [comment]
@komidore64 thanks for the ack, I actually don't have merge permissions :cry: you or @cpeters will have to merge

**Pull request merged**